pages/best-practices/avo-functions-alongside-existing-tracking.mdx

```diff
@@ -0,0 +1,214 @@
+---
+layout: 'guide'
+title: 'Using Avo Functions Alongside Existing Tracking'
+---
+
+import Link from '../../components/Link';
+
+In this guide we'll cover all the required steps to start using Avo in your app or website, alongside existing tracking code and SDKs.
+
+Integrating Avo with your app or website can begin as soon as you create an Avo account, and requires three steps:
+
+1. [Configure Connections in your Avo workspace](#configure-connections-in-your-avo-workspace)
+2. [Install the Avo CLI](#install-the-avo-cli)
+3. [Initialize Avo and send your first event](#initialize-avo-and-send-your-first-event)
+
+## 1. Configure your Avo workspace
+
+In [your Avo workspace](https://avo.app/welcome) head to the Connections tab in the left sidebar. Here we'll create a Source for our app and a Destination for every SDK we want to pass data into.
```

Comment:
If we are talking about using Avo alongside existing tracking wouldn't it be better to talk about a single custom destination from the beginning and not confuse the reader with the option to create multiple destinations?

Reply:
That's a very good point. I'll update to focus on one destination, but add a paragraph how people would go on to implement multiple destinations.

Reply:
Right, I find this notes style with concepts descriptions very nice btw

Reply:
Updated in 4ba6132